src/frontend/components/UI/Anticheat/index.tsx

```diff
@@ -10,30 +10,14 @@ import './index.scss'
 import { useTranslation } from 'react-i18next'
 
 type Props = {
-  gameInfo: GameInfo
+  anticheatInfo: AntiCheatInfo | null
 }
 
 const awacyUrl = 'https://areweanticheatyet.com/'
 
-export default function Anticheat({ gameInfo }: Props) {
+export default function Anticheat({ anticheatInfo }: Props) {
   const { t } = useTranslation()
 
-  const [anticheatInfo, setAnticheatInfo] = useState<AntiCheatInfo | null>(null)
-
-  useEffect(() => {
-    if (
-      gameInfo.runner !== 'sideload' &&
-      gameInfo.title &&
-      gameInfo.namespace !== undefined
-    ) {
-      window.api
-        .getAnticheatInfo(gameInfo.namespace)
-        .then((anticheatInfo: AntiCheatInfo | null) => {
-          setAnticheatInfo(anticheatInfo)
-        })
-    }
-  }, [gameInfo])
```

Comment:
moving this info fetch out and passing it as a propo, so we can see this info in the parent component